### ISSUE TITLE
Uncaught No provider for PlatformRef! error after step 4

### ISSUE THREAD

**Issue by Gdust** [Urigo/angular-meteor#1379]
In browser:
reflective_injector.js:776Uncaught No provider for PlatformRef!ReflectiveInjector_._throwOrNull @ reflective_injector.js:776ReflectiveInjector_._getByKeyDefault @ reflective_injector.js:804ReflectiveInjector_._getByKey @ reflective_injector.js:767ReflectiveInjector_.get @ reflective_injector.js:576createPlatform @ application_ref.js:41MeteorApp.bootstrap @ meteor_app.js:22bootstrap @ bootstrap.js:9meteorInstall.client.app.ts @ app.ts:23fileEvaluate @ install.js:141require @ install.js:75(anonymous function) @ parties.ts:3
In console: 
client/app.ts (19, 16): Argument of type '({ path: string; as: string; component: typeof PartiesList; } | { path: string; as: string; compo...' is not assignable to parameter of type 'RouteDefinition[]'.
  Type '{ path: string; as: string; component: typeof PartiesList; } | { path: string; as: string; compon...' is not assignable to type 'RouteDefinition'.
    Type '{ path: string; as: string; component: typeof PartiesList; }' is not assignable to type 'RouteDefinition'.
      Object literal may only specify known properties, and 'as' does not exist in type 'RouteDefinition'.

I can't continue angular2-meteor tutorial because of this error.

**Comment by NOaMTL** [comment]
check [this](https://github.com/Urigo/angular2-meteor/issues/301)

**Comment by langfors** [comment]
Note: This is the angular1 meteor repository and this issue appears to be with angular2-meteor.  I reported on what worked for me [here](https://github.com/Urigo/angular2-meteor-auto-bootstrap/issues/10).  Hope this helps - if it does, please leave word here so this issue can be closed.

**Comment by Gdust** [comment]
Now I have this errors:
EXCEPTION: Error in /client/app.html:0:0
browser_adapter.js:77 EXCEPTION: Error in /client/app.html:0:0BrowserDomAdapter.logError @ browser_adapter.js:77BrowserDomAdapter.logGroup @ browser_adapter.js:87ExceptionHandler.call @ exception_handler.js:57(anonymous function) @ application_ref.js:301ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283NgZoneImpl.runInner @ ng_zone_impl.js:76NgZone.run @ ng_zone.js:223ApplicationRef_.run @ application_ref.js:290ApplicationRef_.bootstrap @ application_ref.js:312(anonymous function) @ application_ref.js:100ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283(anonymous function) @ angular2-polyfills.js:635ZoneDelegate.invokeTask @ angular2-polyfills.js:423NgZoneImpl.inner.inner.fork.onInvokeTask @ ng_zone_impl.js:36ZoneDelegate.invokeTask @ angular2-polyfills.js:422Zone.runTask @ angular2-polyfills.js:320drainMicroTaskQueue @ angular2-polyfills.js:541(anonymous function) @ dynamics_browser.js:51
browser_adapter.js:77 ORIGINAL EXCEPTION: TypeError: Cannot read property 'annotations' of undefinedBrowserDomAdapter.logError @ browser_adapter.js:77ExceptionHandler.call @ exception_handler.js:66(anonymous function) @ application_ref.js:301ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283NgZoneImpl.runInner @ ng_zone_impl.js:76NgZone.run @ ng_zone.js:223ApplicationRef_.run @ application_ref.js:290ApplicationRef_.bootstrap @ application_ref.js:312(anonymous function) @ application_ref.js:100ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283(anonymous function) @ angular2-polyfills.js:635ZoneDelegate.invokeTask @ angular2-polyfills.js:423NgZoneImpl.inner.inner.fork.onInvokeTask @ ng_zone_impl.js:36ZoneDelegate.invokeTask @ angular2-polyfills.js:422Zone.runTask @ angular2-polyfills.js:320drainMicroTaskQueue @ angular2-polyfills.js:541(anonymous function) @ dynamics_browser.js:51
browser_adapter.js:77 ORIGINAL STACKTRACE:BrowserDomAdapter.logError @ browser_adapter.js:77ExceptionHandler.call @ exception_handler.js:69(anonymous function) @ application_ref.js:301ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283NgZoneImpl.runInner @ ng_zone_impl.js:76NgZone.run @ ng_zone.js:223ApplicationRef_.run @ application_ref.js:290ApplicationRef_.bootstrap @ application_ref.js:312(anonymous function) @ application_ref.js:100ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283(anonymous function) @ angular2-polyfills.js:635ZoneDelegate.invokeTask @ angular2-polyfills.js:423NgZoneImpl.inner.inner.fork.onInvokeTask @ ng_zone_impl.js:36ZoneDelegate.invokeTask @ angular2-polyfills.js:422Zone.runTask @ angular2-polyfills.js:320drainMicroTaskQueue @ angular2-polyfills.js:541(anonymous function) @ dynamics_browser.js:51
browser_adapter.js:77 TypeError: Cannot read property 'annotations' of undefined
    at RouteRegistry.configFromComponent (route_registry.js:113)
    at new RootRouter (router.js:481)
    at routerFactory (router_providers_common.js:24)
    at ReflectiveInjector_._instantiate (reflective_injector.js:697)
    at ReflectiveInjector_._instantiateProvider (reflective_injector.js:626)
    at ReflectiveInjector_._new (reflective_injector.js:615)
    at ReflectiveInjectorDynamicStrategy.getObjByKeyId (reflective_injector.js:270)
    at ReflectiveInjector_._getByKeyDefault (reflective_injector.js:795)
    at ReflectiveInjector_._getByKey (reflective_injector.js:767)
    at ReflectiveInjector_.get (reflective_injector.js:576)BrowserDomAdapter.logError @ browser_adapter.js:77ExceptionHandler.call @ exception_handler.js:70(anonymous function) @ application_ref.js:301ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283NgZoneImpl.runInner @ ng_zone_impl.js:76NgZone.run @ ng_zone.js:223ApplicationRef_.run @ application_ref.js:290ApplicationRef_.bootstrap @ application_ref.js:312(anonymous function) @ application_ref.js:100ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283(anonymous function) @ angular2-polyfills.js:635ZoneDelegate.invokeTask @ angular2-polyfills.js:423NgZoneImpl.inner.inner.fork.onInvokeTask @ ng_zone_impl.js:36ZoneDelegate.invokeTask @ angular2-polyfills.js:422Zone.runTask @ angular2-polyfills.js:320drainMicroTaskQueue @ angular2-polyfills.js:541(anonymous function) @ dynamics_browser.js:51
browser_adapter.js:77 ERROR CONTEXT:BrowserDomAdapter.logError @ browser_adapter.js:77ExceptionHandler.call @ exception_handler.js:73(anonymous function) @ application_ref.js:301ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283NgZoneImpl.runInner @ ng_zone_impl.js:76NgZone.run @ ng_zone.js:223ApplicationRef_.run @ application_ref.js:290ApplicationRef_.bootstrap @ application_ref.js:312(anonymous function) @ application_ref.js:100ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283(anonymous function) @ angular2-polyfills.js:635ZoneDelegate.invokeTask @ angular2-polyfills.js:423NgZoneImpl.inner.inner.fork.onInvokeTask @ ng_zone_impl.js:36ZoneDelegate.invokeTask @ angular2-polyfills.js:422Zone.runTask @ angular2-polyfills.js:320drainMicroTaskQueue @ angular2-polyfills.js:541(anonymous function) @ dynamics_browser.js:51
browser_adapter.js:77 DebugContext {_view: _View_Socially0, _nodeIndex: 0, _tplRow: 0, _tplCol: 0}BrowserDomAdapter.logError @ browser_adapter.js:77ExceptionHandler.call @ exception_handler.js:74(anonymous function) @ application_ref.js:301ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283NgZoneImpl.runInner @ ng_zone_impl.js:76NgZone.run @ ng_zone.js:223ApplicationRef_.run @ application_ref.js:290ApplicationRef_.bootstrap @ application_ref.js:312(anonymous function) @ application_ref.js:100ZoneDelegate.invoke @ angular2-polyfills.js:390NgZoneImpl.inner.inner.fork.onInvoke @ ng_zone_impl.js:45ZoneDelegate.invoke @ angular2-polyfills.js:389Zone.run @ angular2-polyfills.js:283(anonymous function) @ angular2-polyfills.js:635ZoneDelegate.invokeTask @ angular2-polyfills.js:423NgZoneImpl.inner.inner.fork.onInvokeTask @ ng_zone_impl.js:36ZoneDelegate.invokeTask @ angular2-polyfills.js:422Zone.runTask @ angular2-polyfills.js:320drainMicroTaskQueue @ angular2-polyfills.js:541(anonymous function) @ dynamics_browser.js:51
browser_adapter.js:86 EXCEPTION: Error: Uncaught (in promise): EXCEPTION: Error in /client/app.html:0:0
ORIGINAL EXCEPTION: TypeError: Cannot read property 'annotations' of undefined
ORIGINAL STACKTRACE:
TypeError: Cannot read property 'annotations' of undefined
    at RouteRegistry.configFromComponent (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:87383:51)
    at new RootRouter (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:86709:23)
    at routerFactory (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:89128:22)
    at ReflectiveInjector_._instantiate (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50102:27)
    at ReflectiveInjector_._instantiateProvider (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50031:25)
    at ReflectiveInjector_._new (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50020:21)
    at ReflectiveInjectorDynamicStrategy.getObjByKeyId (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:49675:50)
    at ReflectiveInjector_._getByKeyDefault (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50200:38)
    at ReflectiveInjector_._getByKey (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50172:25)
    at ReflectiveInjector_.get (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:49981:21)
ERROR CONTEXT:
[object Object]
browser_adapter.js:77 EXCEPTION: Error: Uncaught (in promise): EXCEPTION: Error in /client/app.html:0:0
ORIGINAL EXCEPTION: TypeError: Cannot read property 'annotations' of undefined
ORIGINAL STACKTRACE:
TypeError: Cannot read property 'annotations' of undefined
    at RouteRegistry.configFromComponent (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:87383:51)
    at new RootRouter (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:86709:23)
    at routerFactory (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:89128:22)
    at ReflectiveInjector_._instantiate (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50102:27)
    at ReflectiveInjector_._instantiateProvider (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50031:25)
    at ReflectiveInjector_._new (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50020:21)
    at ReflectiveInjectorDynamicStrategy.getObjByKeyId (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:49675:50)
    at ReflectiveInjector_._getByKeyDefault (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50200:38)
    at ReflectiveInjector_._getByKey (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50172:25)
    at ReflectiveInjector_.get (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:49981:21)
ERROR CONTEXT:
[object Object]BrowserDomAdapter.logError @ browser_adapter.js:77BrowserDomAdapter.logGroup @ browser_adapter.js:87ExceptionHandler.call @ exception_handler.js:57(anonymous function) @ application_ref.js:265schedulerFn @ async.js:123SafeSubscriber.__tryOrUnsub @ Subscriber.js:225SafeSubscriber.next @ Subscriber.js:174Subscriber._next @ Subscriber.js:124Subscriber.next @ Subscriber.js:88Subject._finalNext @ Subject.js:128Subject._next @ Subject.js:120Subject.next @ Subject.js:77EventEmitter.emit @ async.js:112NgZone._zoneImpl.ng_zone_impl_1.NgZoneImpl.onError @ ng_zone.js:120NgZoneImpl.inner.inner.fork.onHandleError @ ng_zone_impl.js:66ZoneDelegate.handleError @ angular2-polyfills.js:394Zone.runGuarded @ angular2-polyfills.js:300drainMicroTaskQueue @ angular2-polyfills.js:554(anonymous function) @ dynamics_browser.js:51
browser_adapter.js:77 STACKTRACE:BrowserDomAdapter.logError @ browser_adapter.js:77ExceptionHandler.call @ exception_handler.js:59(anonymous function) @ application_ref.js:265schedulerFn @ async.js:123SafeSubscriber.__tryOrUnsub @ Subscriber.js:225SafeSubscriber.next @ Subscriber.js:174Subscriber._next @ Subscriber.js:124Subscriber.next @ Subscriber.js:88Subject._finalNext @ Subject.js:128Subject._next @ Subject.js:120Subject.next @ Subject.js:77EventEmitter.emit @ async.js:112NgZone._zoneImpl.ng_zone_impl_1.NgZoneImpl.onError @ ng_zone.js:120NgZoneImpl.inner.inner.fork.onHandleError @ ng_zone_impl.js:66ZoneDelegate.handleError @ angular2-polyfills.js:394Zone.runGuarded @ angular2-polyfills.js:300drainMicroTaskQueue @ angular2-polyfills.js:554(anonymous function) @ dynamics_browser.js:51
browser_adapter.js:77 Error: Uncaught (in promise): EXCEPTION: Error in /client/app.html:0:0
ORIGINAL EXCEPTION: TypeError: Cannot read property 'annotations' of undefined
ORIGINAL STACKTRACE:
TypeError: Cannot read property 'annotations' of undefined
    at RouteRegistry.configFromComponent (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:87383:51)
    at new RootRouter (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:86709:23)
    at routerFactory (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:89128:22)
    at ReflectiveInjector_._instantiate (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50102:27)
    at ReflectiveInjector_._instantiateProvider (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50031:25)
    at ReflectiveInjector_._new (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50020:21)
    at ReflectiveInjectorDynamicStrategy.getObjByKeyId (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:49675:50)
    at ReflectiveInjector_._getByKeyDefault (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50200:38)
    at ReflectiveInjector_._getByKey (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50172:25)
    at ReflectiveInjector_.get (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:49981:21)
ERROR CONTEXT:
[object Object]
    at resolvePromise (http://localhost:3000/packages/barbatus_angular2-runtime.js?hash=fda9b73362c52e988ad030102a9f58e4d584cda3:4898:39)
    at http://localhost:3000/packages/barbatus_angular2-runtime.js?hash=fda9b73362c52e988ad030102a9f58e4d584cda3:4932:25
    at ZoneDelegate.invokeTask (http://localhost:3000/packages/barbatus_angular2-runtime.js?hash=fda9b73362c52e988ad030102a9f58e4d584cda3:4721:174)
    at Object.NgZoneImpl.inner.inner.fork.onInvokeTask (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:54164:41)
    at ZoneDelegate.invokeTask (http://localhost:3000/packages/barbatus_angular2-runtime.js?hash=fda9b73362c52e988ad030102a9f58e4d584cda3:4721:72)
    at Zone.runTask (http://localhost:3000/packages/barbatus_angular2-runtime.js?hash=fda9b73362c52e988ad030102a9f58e4d584cda3:4634:55)
    at drainMicroTaskQueue (http://localhost:3000/packages/barbatus_angular2-runtime.js?hash=fda9b73362c52e988ad030102a9f58e4d584cda3:4835:43)
    at http://localhost:3000/packages/meteor.js?hash=ae8b8affa9680bf9720bd8f7fa112f13a62f71c3:1105:22BrowserDomAdapter.logError @ browser_adapter.js:77ExceptionHandler.call @ exception_handler.js:60(anonymous function) @ application_ref.js:265schedulerFn @ async.js:123SafeSubscriber.__tryOrUnsub @ Subscriber.js:225SafeSubscriber.next @ Subscriber.js:174Subscriber._next @ Subscriber.js:124Subscriber.next @ Subscriber.js:88Subject._finalNext @ Subject.js:128Subject._next @ Subject.js:120Subject.next @ Subject.js:77EventEmitter.emit @ async.js:112NgZone._zoneImpl.ng_zone_impl_1.NgZoneImpl.onError @ ng_zone.js:120NgZoneImpl.inner.inner.fork.onHandleError @ ng_zone_impl.js:66ZoneDelegate.handleError @ angular2-polyfills.js:394Zone.runGuarded @ angular2-polyfills.js:300drainMicroTaskQueue @ angular2-polyfills.js:554(anonymous function) @ dynamics_browser.js:51
angular2-polyfills.js:528 Unhandled Promise rejection: EXCEPTION: Error in /client/app.html:0:0
ORIGINAL EXCEPTION: TypeError: Cannot read property 'annotations' of undefined
ORIGINAL STACKTRACE:
TypeError: Cannot read property 'annotations' of undefined
    at RouteRegistry.configFromComponent (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:87383:51)
    at new RootRouter (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:86709:23)
    at routerFactory (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:89128:22)
    at ReflectiveInjector_._instantiate (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50102:27)
    at ReflectiveInjector_._instantiateProvider (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50031:25)
    at ReflectiveInjector_._new (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50020:21)
    at ReflectiveInjectorDynamicStrategy.getObjByKeyId (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:49675:50)
    at ReflectiveInjector_._getByKeyDefault (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50200:38)
    at ReflectiveInjector_._getByKey (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:50172:25)
    at ReflectiveInjector_.get (http://localhost:3000/packages/modules.js?hash=c27c0e46291dabb99bee9797664334b35c04ada6:49981:21)
ERROR CONTEXT:
[object Object] ; Zone: angular ; Task: Promise.then ; Value: ViewWrappedException {_wrapperMessage: "Error in /client/app.html:0:0", _originalException: InstantiationError, _originalStack: undefined, _context: DebugContext, _wrapperStack: "Error: Error in /client/app.html:0:0↵    at ViewWr…c27c0e46291dabb99bee9797664334b35c04ada6:53743:26"}consoleError @ angular2-polyfills.js:528drainMicroTaskQueue @ angular2-polyfills.js:557(anonymous function) @ dynamics_browser.js:51
angular2-polyfills.js:530 Error: Uncaught (in promise): EXCEPTION: Error in /client/app.html:0:0(…)

**Comment by Urigo** [comment]
the tutorial was updated and deployed.
Please open another issue if it doesn't work for you
